test/e2e/storage/external/external.go

```diff
@@ -306,6 +307,12 @@ func (d *driverDefinition) GetDynamicProvisionStorageClass(e2econfig *storagefra
 		// reconsidered if we eventually need to move in-tree storage tests out.
 		sc.Parameters["csi.storage.k8s.io/fstype"] = fsType
 	}
+
+	// Remove the default annotation from the storage class if they exists.
+	// Multiple storage classes with this annotation will result in failure.
+	delete(sc.Annotations, storageutil.BetaIsDefaultStorageClassAnnotation)
+	delete(sc.Annotations, storageutil.IsDefaultStorageClassAnnotation)
+
 	return storageframework.CopyStorageClass(sc, f.Namespace.Name, "e2e-sc")
```

Comment:
Why don't we remove the annotation in `CopyStorageClass()` to remove from the copied StorageClass?

Reply:
Good one. Moved it in there.